export const newAbortError = () =>
  new DOMException('This operation was aborted', 'AbortError')
